### PR TITLE
Rewrite the intro to IOTA.

### PR DESCRIPTION
**Pull request by aleksei-korolev**
# Description of change

I've rewritten the intro to IOTA to make it more approachable for new users. I've added a bit of a guidance around our ecosystem and I've split the original article into two and moved it down the table of content.

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

- Documentation Enhancement

## How the change has been tested

I've read it out loud.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [X] I have followed the contribution guidelines for this project
- [X] I have performed a self-review of my own code
- [ ] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

**Comment by aleksei-korolev** [comment]
@lucas-tortora @Phyloiota

File: internal/learn/about-iota/an-introduction-to-iota.md [hunk 1]
```diff
@@ -1,74 +1,15 @@
...
-  the Ecosystem developed around it.
+title: IOTA
+description: A basic introduction to the IOTA protocol.
 ---
 
-# An Introduction to IOTA
+# IOTA
 
-**This topic gives a brief overview and describes some of the main differences between IOTA's Tangle and a blockchain.**
+IOTA is a distributed ledger technology, a DLT. With the right approach, DLTs let us control our own [private data](https://wiki.iota.org/identity.rs/introduction), run [programs](https://wiki.iota.org/smart-contracts/overview) that nobody can meddle with, and trade and own assets without any middlemen or middlebanks.
```
Comment (lucas-tortora) [inline internal/learn/about-iota/an-introduction-to-iota.md:9]:
```suggestion
IOTA is a distributed ledger technology, a DLT. With the right approach, DLTs let us control our own [private data](https://wiki.iota.org/identity.rs/introduction), run [programs](https://wiki.iota.org/smart-contracts/overview) that nobody can meddle with, and trade and own assets without any intermediaries.
```

File: internal/learn/about-iota/an-introduction-to-iota.md [hunk 2]
```diff
@@ -1,74 +1,15 @@
...
 
-Blockchains and the Tangle both fall under the same category of distributed ledger technology (DLT).
+As in the name, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if all nodes were controlled by the same entity, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node joins the network, so any DLT needs a way to protect itself. The way that IOTA takes makes it differ from other protocols.
```
Comment (lucas-tortora) [inline internal/learn/about-iota/an-introduction-to-iota.md:11]:
```suggestion
As in the name, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if the same entity controlled all nodes, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node will join the network, so any DLT needs a way to protect itself. IOTA approaches this differently than protocols.
```

File: internal/learn/about-iota/an-introduction-to-iota.md [hunk 3]
```diff
@@ -1,74 +1,15 @@
 ---
 id: an-introduction-to-iota
-title: An Introduction to IOTA
-description: A basic introduction to the IOTA DLT Technology. The use-cases and
-  the Ecosystem developed around it.
+title: IOTA
+description: A basic introduction to the IOTA protocol.
```
Comment (lucas-tortora) [inline internal/learn/about-iota/an-introduction-to-iota.md:4]:
```suggestion
description: IOTA runs on the Tangle, a structure where newer transactions verify the older ones, and pretty much any other DLT runs on a blockchain instead.
```
We should also add an image tag to these files  so there's a nice thumbnail if they're shared :)   Off the top of my head a screenshot from the gifs could work, or a nice IOTA logo.

File: internal/learn/about-iota/an-introduction-to-iota.md [hunk 4]
```diff
@@ -1,74 +1,15 @@
...
-
-To allow the nodes to recognize valid milestones, all nodes that participate in the same IOTA network are configured with the Ed25519 signatures of a coordinator they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted Coordinator did sign them.
-
-To ensure that new messages always have a chance of being confirmed, the Coordinator sends indexed milestones signed with those signatures regularly (every 10 seconds). This way ensures that nodes can compare the indexes of their milestones to check whether they are synchronized with the rest of the network.
-
-![coordinator](/img/learn/milestones.gif)
+The IOTA protocol is still in research. It has two public networks: the IOTA mainnet is the stable network that manages your IOTA tokens, and the Shimmer testnet is the staging network for the latest and greatest protocol updates. Once the changes are well tested on Shimmer, they come to the mainnet. The next big update is [Stardust](https://blog.shimmer.network/stardust-upgrade-in-a-nutshell/). The future update that concludes decentralization effort is called [Coordicide](roadmap-to-decentralization.md).
```
Comment (lucas-tortora) [inline internal/learn/about-iota/an-introduction-to-iota.md:15]: could you add links to IOTA mainnet and Shimmer ?

File: internal/learn/about-iota/coordinator.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+id: coordinator
+title: Coordinator
+description: What is Coordinator in the IOTA network.
+---
+
+# Coordinator
+
+:::note
+
+Coordinator is a temporary solution. IOTA will remove Coordinator in the future [IOTA 2.0 update](./roadmap-to-decentralization.md).
```
Comment (lucas-tortora) [inline internal/learn/about-iota/coordinator.md:11]:
```suggestion
The Coordinator is a temporary solution. IOTA will remove the Coordinator in the future [IOTA 2.0 update](./roadmap-to-decentralization.md).
```

File: internal/learn/about-iota/an-introduction-to-iota.md (hunk 3 above)
Comment (lucas-tortora) [inline internal/learn/about-iota/an-introduction-to-iota.md:4]:
```suggestion
description: IOTA runs on the Tangle, a structure where newer transactions verify the older ones
```

These articles could do with an `image` tag as well so there's a nice thumbnail if someone shares the link. 

Off the top of my head, it could be a screenshot from one of the gifs, or an IOTA logo.

File: internal/learn/about-iota/an-introduction-to-iota.md (hunk 1 above)
Comment (lucas-tortora) [inline internal/learn/about-iota/an-introduction-to-iota.md:9]:
```suggestion
IOTA is a distributed ledger technology, a DLT. With the right approach, DLTs let us control our own [private data](https://wiki.iota.org/identity.rs/introduction), run [programs](https://wiki.iota.org/smart-contracts/overview) that nobody can meddle with, and trade and own assets without any intermediaries.
```

File: internal/learn/about-iota/an-introduction-to-iota.md (hunk 2 above)
Comment (lucas-tortora) [inline internal/learn/about-iota/an-introduction-to-iota.md:11]:
```suggestion
As the name implies, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if the same entity controlled all nodes, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node will join the network, so any DLT needs a way to protect itself. IOTA approaches this differently than other protocols. 
```

File: internal/learn/about-iota/an-introduction-to-iota.md [hunk 6]
```diff
@@ -1,74 +1,15 @@
...
 
-Blockchains and the Tangle both fall under the same category of distributed ledger technology (DLT).
+As in the name, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if all nodes were controlled by the same entity, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node joins the network, so any DLT needs a way to protect itself. The way that IOTA takes makes it differ from other protocols.
 
-The main differences between blockchains and the Tangle are the following:
+IOTA runs on [the Tangle](./tangle.md), a structure where newer transactions verify the older ones, and pretty much any other DLT runs on a blockchain instead. To secure its state and history, a blockchain has to collect transactions in blocks and chain the blocks one after another. That leads to a natural bottleneck: imagine loading the world's cargo into a single train wagon by wagon. IOTA bypasses it altogether.
```
Comment (lucas-tortora) [inline internal/learn/about-iota/an-introduction-to-iota.md:13]:
```suggestion
IOTA runs on [the Tangle](./tangle.md), a structure where newer transactions verify the older ones, and pretty much any other DLT runs on a blockchain instead. To secure its state and history, a blockchain must collect transactions in blocks and chain the blocks one after another. That leads to a natural bottleneck: imagine loading the world's cargo into a single train wagon by wagon. IOTA bypasses it altogether.
```

File: internal/learn/about-iota/coordinator.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+---
+id: coordinator
+title: Coordinator
```
Comment (lucas-tortora) [inline internal/learn/about-iota/coordinator.md:3]:
```suggestion
title: The Coordinator
```

File: internal/learn/about-iota/coordinator.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+description: What is Coordinator in the IOTA network.
+---
+
+# Coordinator
```
Comment (lucas-tortora) [inline internal/learn/about-iota/coordinator.md:7]:
```suggestion
# The Coordinator
```

File: internal/learn/about-iota/coordinator.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+:::note
+
+Coordinator is a temporary solution. IOTA will remove Coordinator in the future [IOTA 2.0 update](./roadmap-to-decentralization.md).
+
+:::
+
+Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.
```
Comment (lucas-tortora) [inline internal/learn/about-iota/coordinator.md:15]:
```suggestion
The Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are only considered for confirmation when they are directly or indirectly referenced by a milestone that nodes have validated.
```

File: internal/learn/about-iota/coordinator.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
+---
+id: coordinator
+title: Coordinator
+description: What is Coordinator in the IOTA network.
```
Comment (lucas-tortora) [inline internal/learn/about-iota/coordinator.md:4]:
```suggestion
description: The Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages.
```
and please add an image as well

File: internal/learn/about-iota/coordinator.md (hunk 9 above)
Comment (lucas-tortora) [reply internal/learn/about-iota/coordinator.md:15]: this could use a link to What is Chrysalis , https://wiki.iota.org/chrysalis-docs/introduction/what_is_chrysalis#new-milestone-selection-algorithm

File: internal/learn/about-iota/coordinator.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+:::note
+
+Coordinator is a temporary solution. IOTA will remove Coordinator in the future [IOTA 2.0 update](./roadmap-to-decentralization.md).
+
+:::
+
+Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.
+
+To allow the nodes to recognize valid milestones, all nodes that participate in the same IOTA network are configured with the Ed25519 signatures of a coordinator they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted coordinator did sign them.
```
Comment (lucas-tortora) [inline internal/learn/about-iota/coordinator.md:17]:
```suggestion
To allow the nodes to recognize valid milestones, all nodes participating in the same IOTA network are configured with the [Ed25519 signatures](https://wiki.iota.org/chrysalis-docs/introduction/what_is_chrysalis#ed25519-signature-scheme) of a coordinator they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted coordinator did sign them.
```

File: internal/learn/about-iota/coordinator.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+
+To ensure that new messages always have a chance of being confirmed, Coordinator sends indexed milestones signed with those signatures regularly (every 10 seconds). This way ensures that nodes can compare the indexes of their milestones to check if they are synchronized with the rest of the network.
```
Comment (lucas-tortora) [inline internal/learn/about-iota/coordinator.md:19]:
```suggestion
To ensure that new messages always have a chance of being confirmed, the Coordinator sends indexed milestones signed with those signatures regularly (every 10 seconds). This ensures that nodes can compare the indexes of their milestones to check if they are synchronized with the rest of the network.
```

File: internal/learn/about-iota/tangle.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
+---
+id: tangle
+title: The Tangle
+description: The Tangle is the data structure behind IOTA and it is different from a blockchain.
+---
```
Comment (lucas-tortora) [inline internal/learn/about-iota/tangle.md:5]: this could do with an image

File: internal/learn/about-iota/tangle.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+# The Tangle
+
+The Tangle is a structure that holds all necessary information to keep track of token ownership. It is a directed acyclic graph (DAG) of messages, where each newer message is attached to two to eight older ones. The protocol can process these various attachments in parallel.
```
Comment (lucas-tortora) [inline internal/learn/about-iota/tangle.md:9]:
```suggestion
The Tangle is a structure that holds all necessary information to keep track of token ownership. It is a directed acyclic graph (DAG) of messages, where each newer message is attached to two to eight older ones. The protocol can process these various attachments in parallel.
```

**Review by lucas-tortora — CHANGES_REQUESTED** [review]

File: internal/learn/about-iota/an-introduction-to-iota.md (hunk 4 above)
Comment (aleksei-korolev) [reply internal/learn/about-iota/an-introduction-to-iota.md:15]: I've added for the ../future/shimmer.md, but I'm not sure what to reference for the mainnet.

File: internal/learn/about-iota/coordinator.md (hunk 9 above)
Comment (aleksei-korolev) [reply internal/learn/about-iota/coordinator.md:15]: I don't like it. Crysalis is the reality for the long time now and the article still talks about it as of a new feature. I believe that not a single newcomer cares about the fact the current revision of IOTA is called Crysalis or how awesome it used to be compared to IOTA 1.0.

File: internal/learn/about-iota/coordinator.md (hunk 11 above)
Comment (aleksei-korolev) [reply internal/learn/about-iota/coordinator.md:17]: I've removed the mention of Ed25519. Same concern as with crysalis in general.

File: internal/learn/about-iota/tangle.md [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+
+*The blockchain bottleneck.*
+
+Transactions in a blockchain can only become part of the ledger if they are included in a newly issued block by the block producer (a miner in Proof of Work blockchains, a staker in Proof of Stake blockchains). In nearly all blockchains, the block producers can decide which new transactions they include in their blocks. They of course favor users willing to pay higher fees for their transactions. Any time there are more transactions that can pass through the blockchain bottleneck, the transaction fees ramp up.
+
+There is more. In blockchains, each block producer works on a potential new block, but only a single producer will become the leader of the blockchain and get its block attached. All other potential blocks will be discarded — and in case of PoW systems, all effort and electricity put into computing them will be wasted. 
+
+Only the blockchain leader collects the reward for producing a block. Only the largest miners and stakers can collect this reward consistently. It promotes a system with fewer but bigger players, which can be perceived as a form of centralization.
```
[the remark this answers is not in the document]
Comment (aleksei-korolev) [reply internal/learn/about-iota/tangle.md:27]: I'm not that sure about spiitting it into subsections. All topics are so interrelated and the section goes full circle by showing the tangle jarneck. I think the animations break up the text good enough for the weaker structure to suffice.

[the remark this answers is not in the document]
Comment (lucas-tortora) [reply internal/learn/about-iota/tangle.md:30]: I think the titles would help break up the text a bit, mainly for people who are skimming over the article.

**Review by lucas-tortora — APPROVED** [review]
LGTM!

File: internal/learn/about-iota/an-introduction-to-iota.md (hunk 2 above)
Comment (Phyloiota) [reply internal/learn/about-iota/an-introduction-to-iota.md:14]: One suggestion from my side, remove the mention of " a DLT maintains a SINGLE ledger. Many DLTs handle multiple ledgers and sidechains / parachains. IOTA will do the same, so at one point it will become misleading. A DLT can manage multiple networks, of which each handles a single ledger of records of token ownership

File: internal/learn/about-iota/an-introduction-to-iota.md [hunk 16]
```diff
@@ -1,74 +1,18 @@
...
-
-To ensure that new messages always have a chance of being confirmed, the Coordinator sends indexed milestones signed with those signatures regularly (every 10 seconds). This way ensures that nodes can compare the indexes of their milestones to check whether they are synchronized with the rest of the network.
-
-![coordinator](/img/learn/milestones.gif)
+The IOTA protocol is still in research. It has two public networks: the IOTA mainnet is the stable network that manages your IOTA tokens, and the [Shimmer testnet](../future/shimmer.md) is the staging network for the latest and greatest protocol updates. Once the changes are well tested on Shimmer, they come to the mainnet. The next big update is [Stardust](https://blog.shimmer.network/stardust-upgrade-in-a-nutshell/). The future update that concludes decentralization effort is called [Coordicide](roadmap-to-decentralization.md).
```
Comment (Phyloiota) [inline internal/learn/about-iota/an-introduction-to-iota.md:18]: do not call it Shimmer testnet! It is not!

File: internal/learn/about-iota/tangle.md [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+image: /img/learn.svg
+keywords:
+- explanation
+---
+
+# The Tangle
+
+The Tangle is a structure that holds all necessary information to keep track of token ownership. It is a directed acyclic graph (DAG) of messages, where each newer message is attached to two to eight older ones. The protocol can process these various attachments in parallel.
+
+IOTA nodes reach consensus on the ledger state by validating massages. Anyone can send a message [for free](./why-is-iota-feeless.md). Currently, messages will only be considered valid if they reference a milestone. Milestones are issued by a special network node that is called [Coordinator](./coordinator.md).
```
Comment (Phyloiota) [inline internal/learn/about-iota/tangle.md:14]:
```suggestion
IOTA nodes reach consensus on the ledger state by validating blocks. Anyone can send a block [for free](./why-is-iota-feeless.md). Currently, blocks will only be considered valid if they reference a milestone. Milestones are issued by a special network node that is called [Coordinator](./coordinator.md).
```

File: internal/learn/about-iota/tangle.md [hunk 18]
```diff
@@ -0,0 +1,42 @@
...
+title: The Tangle
+description: The Tangle is the data structure behind IOTA and it is different from a blockchain.
+image: /img/learn.svg
+keywords:
+- explanation
+---
+
+# The Tangle
+
+The Tangle is a structure that holds all necessary information to keep track of token ownership. It is a directed acyclic graph (DAG) of messages, where each newer message is attached to two to eight older ones. The protocol can process these various attachments in parallel.
```
Comment (Phyloiota) [inline internal/learn/about-iota/tangle.md:12]:
```suggestion
The Tangle is a structure that holds all necessary information to keep track of token ownership. It is a directed acyclic graph (DAG) of blocks, where each newer block is attached to two to eight older ones. Every block contains only a single change to the ledger state. The protocol can process these various attachments in parallel.
```

File: internal/learn/about-iota/an-introduction-to-iota.md (hunk 2 above)
Comment (Phyloiota) [reply internal/learn/about-iota/an-introduction-to-iota.md:14]: And we have to have a strategy how to address the new nomenklatur. We decided to name everything blocks now.... How is this handled here? Messages should be completely removed from the wiki

**Review by Phyloiota — COMMENTED** [review]
I think we need to clarify mostly how we translate the concepts of "blocks" into the documentation... I think we decided to not put it in all public-facing docs, but also have decided to move completely away from the term "messages" It would be best to clarify this first and then change it everywhere

**Comment by aleksei-korolev** [comment]
As discussed, let's keep messages as messages when we talk about IOTA 1.5.

File: internal/learn/about-iota/coordinator.md [hunk 19]
```diff
@@ -0,0 +1,26 @@
...
+
+# The Coordinator
+
+:::note
+
+The Coordinator is a temporary solution. IOTA will remove the Coordinator in the future [IOTA 2.0 update](./roadmap-to-decentralization.md).
+
+:::
+
+Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.
```
Comment (jlvandenhout) [inline internal/learn/about-iota/coordinator.md:18]:
```suggestion
The Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.
```

File: internal/learn/about-iota/coordinator.md [hunk 20]
```diff
@@ -0,0 +1,26 @@
...
+
+To allow the nodes to recognize valid milestones, all IOTA nodes on the same network are configured with the signatures of a coordinator node which they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted coordinator did sign them.
```
Comment (jlvandenhout) [inline internal/learn/about-iota/coordinator.md:20]:
```suggestion
To allow the nodes to recognize valid milestones, all IOTA nodes on the same network are configured with the signatures of a coordinator node which they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted Coordinator did sign them.
```

File: internal/learn/about-iota/tangle.md [hunk 21]
```diff
@@ -0,0 +1,42 @@
...
+
+There is more. In blockchains, each block producer works on a potential new block, but only a single producer will become the leader of the blockchain and get its block attached. All other potential blocks will be discarded — and in case of PoW systems, all effort and electricity put into computing them will be wasted. 
+
+Only the blockchain leader collects the reward for producing a block. Only the largest miners and stakers can collect this reward consistently. It promotes a system with fewer but bigger players, which can be perceived as a form of centralization.
+
+### The Tangle
+
+The Tangle is a network of parallel processed transactions. These parallel transactions form the "front" of the Tangle and offer multiple points for newly issued transactions to attach to. There are no block producers, and it is enough to issue your transaction to any node to imprint into the Tangle.
```
Comment (jlvandenhout) [inline internal/learn/about-iota/tangle.md:36]: > There are no block producers, and it is enough to issue your transaction to any node to imprint into the Tangle.

Not sure if this is correct? Isn't the issuer it's own block producer (or message producer in Chrysalis terms)?

Also the use of "imprint" feels a bit confusing as a non-native speaker. Could we rephrase it to use previously used terminology like "attach to the Tangle" or something?

File: internal/learn/about-iota/tangle.md [hunk 22]
```diff
@@ -0,0 +1,42 @@
...
+
+IOTA is a leaderless protocol. Multiple nodes attach multiple transactions to multiple points of the Tangle at the same time. IOTA has no bottlenecks, no middlemen, and no fees.
```
Comment (jlvandenhout) [inline internal/learn/about-iota/tangle.md:38]: This may be a more general comment, but: we seem to use "transaction" and "message" interchangeably in this text (and I think in others as well). To my understanding transactions are a type of payload which are added to a message and messages get attached to the Tangle (just like in blockchains transactions get added to a block and the block gets added to the chain).

We have messages with other payloads as well, like indexation messages. Wouldn't it be better to make this distinction early on to avoid confusion? Or at least be consistent in the use of those terms?

**Review by jlvandenhout — CHANGES_REQUESTED** [review]

**Review by jlvandenhout — APPROVED** [review]
